contributing/core-contributor-getting-started.md

```diff
@@ -0,0 +1,27 @@
+# Getting started with being a core contributor to MDC iOS
+
+Checklist:
+
+- [ ] [Install git-clang-format](#git-clang-format).
+- [ ] Install https://github.com/material-foundation/github-squash-and-merge-pr-descriptions. This ensures that your pull request's merge commit matches your PR description.
+
+## Installation guides
+
+### git-clang-format
+
+```bash
+# First check to see if you have clang-format installed.
+git clang-format -h
+
+# Download the script
+curl -s https://raw.githubusercontent.com/llvm-mirror/clang/master/tools/clang-format/git-clang-format -o "git-clang-format"
```

Comment:
Is this secure? Do we need to check against a known hash or something?

Reply:
Done.